app/helpers/fact_values_helper.rb

```diff
@@ -60,7 +60,7 @@ def fact_breadcrumbs
           url: (fact_values_path if authorized_for(hash_for_fact_values_path)),
         },
         {
-          caption: Setting[:display_fqdn_for_hosts] ? params[:host_id].split('.')[0] : params[:host_id],
+          caption: host_display_name(params[:host_id]),
```

Comment:
@sbernhard We can't use the `to_label` here as we don't have the `@host` model here.

Reply:
I thought, if we can use the new host_display_name at both locations, fact_values _helper and model/host/base

Reply:
I don't think mixing helpers into models is a good idea. AFAIK helpers are for the views, not for the models.

Reply:
Ah. Yes, now I remember. Sometimes it would be very helpful to have helper methods which can be used in views, models and controllers. Maybe to define such helpers in foreman/lib/ ?